Neos.Neos/Classes/Security/ManagableRolesCalculatorInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the Neos.Neos package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+namespace Neos\Neos\Security;
+
+use Neos\Neos\Domain\Model\User;
+
+interface ManagableRolesCalculatorInterface
```

Comment:
Why do we need an interface for this?
I ask because In the past we have added interfaces to parts that weren't really meant to be extensible and later had to remove them again (= breaking change).